src/test/suite/connectionController.test.ts

```diff
@@ -743,17 +739,20 @@ suite('Connection Controller Test Suite', function () {
       name: 'localhost:27017',
       storageLocation: StorageLocation.GLOBAL,
       connectionModel: {
+        _id: '1',
+        isFavorite: false,
+        name: 'Local 1',
         isSrvRecord: false,
         hostname: 'localhost',
         port: 27017,
         hosts: [{ host: 'localhost', port: 27017 }],
         extraOptions: {},
         connectionType: 'NODE_DRIVER',
-        authStrategy: 'NONE',
-        readPreference: 'primary',
+        authStrategy: AUTH_STRATEGY_VALUES.NONE,
+        readPreference: READ_PREFERENCES.PRIMARY,
```

Comment:
Why did these changes become necessary? I’d really like to go the other direction if we can with union types, unless there’s some real value in the enums here which I’m missing

Reply:
We use these enums in LegacyConnectionModel type, I removed ConnectionModel in this PR and replaced it with LegacyConnectionModel

Reply:
I’m not sure I understand fully – in `LegacyConnectionModel`, these are also string constant union types:

https://github.com/mongodb-js/compass/blob/0c41452ccc576bfbcc77f445a5a26b2ef289b4f2/packages/data-service/src/legacy/legacy-connection-model.ts#L37-L43

Reply:
Oh wait, I missed that the vscocde extension has its own `LegacyConnectionModel` … nvm then :)